.github/workflows/frontend-lint.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Frontend linting
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
Current `node` maintained version is `14.18`. Please see [tool-versions](https://github.com/vrn-sh/erp/blob/18-set-up-front-end-ci/.tool-versions) & [asdf](https://asdf-vm.com/guide/getting-started.html) for more info